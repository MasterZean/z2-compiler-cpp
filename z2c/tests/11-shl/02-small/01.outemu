Small{96}
Small{96}
Small{96}
Small{96}
Small{96}
Small{96}
Small{96}
Small{96}
Small{96}
