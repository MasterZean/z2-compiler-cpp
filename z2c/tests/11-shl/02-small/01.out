70 << 4;
70 << 4u;
70 << 4;
70 << 4u;
70 << 4;
70 << 4u;
70 << 4ll;
70 << 4ull;
70 << 4u;
DummyClass(1, 6): error:
	can't apply operator '<<' on types: 
		'Small' and 
		'Bool'
DummyClass(2, 6): error:
	can't apply operator '<<' on types: 
		'Small' and 
		'Bool'
DummyClass(11, 6): error:
	can't apply operator '<<' on types: 
		'Small' and 
		'Float'
DummyClass(12, 6): error:
	can't apply operator '<<' on types: 
		'Small' and 
		'Double'
DummyClass(13, 6): error:
	can't apply operator '<<' on types: 
		'Small' and 
		'Char'
